firmware/openbot_ble/openbot_ble.ino

```diff
@@ -0,0 +1,1856 @@
+// Required App Version: 0.5
```

Comment:
How many changes are required to the firmware? I think it would make more sense to integrate it with the existing firmware and use a flag such as #HAS_BLUETOOTH to turn the feature on/off in the firmware.

Reply:
Hi, I've merged the both BLE with the BLUETOOTH flag.